apps/docs/content/components/Pillar/react.mdx

```diff
@@ -67,6 +67,20 @@ You can add an image to enhance the visual context.
 </Pillar>
 ```
 
+### Border
+
+You can add an optional border to create a visual boundary around the pillar content.
```

Comment:
@jesussandreas Is this okay or do you want to elaborate on why / when users should apply a border?

Reply:
This should be fine for the React docs, but we may want to update the interface guidelines to include some guidance on when to apply borders. We can address this as a follow-up.